config/settings.rb

```diff
@@ -6,8 +6,12 @@
 
 SETTINGS.merge! YAML.load(ERB.new(File.read("#{root}/#{settings_file}")).result) if File.exist?(settings_file)
 SETTINGS[:version] = Foreman::Version.new
-SETTINGS[:unattended] = SETTINGS[:unattended].nil? || SETTINGS[:unattended]
-SETTINGS[:login]    ||= SETTINGS[:login].nil? || SETTINGS[:ldap]
+
+# default to true if missing
+[:unattended, :login, :locations_enabled, :organizations_enabled].each do |setting|
+  SETTINGS[setting] = SETTINGS[setting].nil? || SETTINGS[setting]
```

Comment:
should we create a role where unattended is disabled? (e.g disabled by permissions)